tests/test_prometheus.py

```
@@ -0,0 +1,127 @@
+
+@unittest.skipUnless(prometheus_client, 'prometheus_client is not installed')
+class _RequiresPrometheusClient(unittest.TestCase):
+    pass
+
+
+class PrometheusConfigTestCase(unittest.TestCase):
+    def test_address_defaults_to_localhost(self):
+        self.assertEqual(models.PrometheusConfig().address, '127.0.0.1')
+
+    def test_address_configurable(self):
+        config = models.PrometheusConfig(address='0.0.0.0')
```

Ruff (0.15.20)

[error] Possible binding to all interfaces

(S104)